cw_monero/lib/transaction_history.dart

```diff
@@ -145,3 +216,15 @@ Future<PendingTransactionDescription> createTransaction(
       'priorityRaw': priorityRaw,
       'accountIndex': accountIndex
     });
+
+Future<PendingTransactionDescription> createTransactionMultDest(
+    {List<MoneroOutput> outputs,
+      String paymentId,
```

Comment:
PaymentId is kind of deprecated, so better to make keep it optional with empty string by default.